### DISCUSSION TITLE
ESM-2 pretraining dataset

### DISCUSSION THREAD

**Discussion by jasperhyp** [facebookresearch/esm#388] [General] [answered]
Hi!  I noticed in the paper that you used UniRef Sep 2021 release.  But in their FTP (https://ftp.uniprot.org/pub/databases/uniprot/previous_releases/) there is no 2021-09 release.  Can you clarify?  Thanks!

**Comment by halilakin** [comment, marked as answer]
Hi. The dataset we used was created from [2021_04](https://ftp.uniprot.org/pub/databases/uniprot/previous_releases/release-2021_04/) release. Sorry for the confusion here.

**Reply by jasperhyp** [reply]
Thanks!  Is the train/val split for this dataset uploaded?  I only saw the split for the 2018 release used for ESM-1b.  If I am not mistaken, the reported numbers of ESM-2 are on the val set, so I wanted to keep everything consistent!

**Comment by robert-verkuil** [comment]
(Adding some details re: the release date)
Link we referenced when writing "September 2021": [Link](https://ftp.uniprot.org/pub/databases/uniprot/previous_releases/release-2021_04/knowledgebase/UniProtKB_SwissProt-relstat.html)
Just checked relnotes.txt, and it says "November 2021": [Link](https://ftp.uniprot.org/pub/databases/uniprot/previous_releases/release-2021_04/relnotes.txt)

Either way it should be the 2021_04 release Halil's links above. 👍 
(YYYY_NN seems to denote (year, release number), not (year, month).)

**Reply by jasperhyp** [reply]
Thank you!  I know I'm asking for a lot but just one question more, is there possibly a fasta UniRef90 or UniRef50 that can be shared?  The previous releases only contain several hundred GB+ XML files, which could take quite a while to parse...
